script.js

```diff
@@ -0,0 +1,3 @@
+const orderQueue = ['1XT567437','1U7857317','1I9222528'];
+let shipping = orderQueue.shift();
```

Comment:
Can we change this to `const` since this variable is not reassigned?
```suggestion
const shipping = orderQueue.shift();
```